themes/mcf/components/FooterComponents/SocialMediaContent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { ExternalLink } from "@components/ExternalLink";
+import { FacebookIcon, InstagramIcon, LinkedInIcon, TwitterIcon } from "@components/svg/Icons";
+
+const SocialMediaContent = () => {
+  const multilateralClimateFundSocialMedia = {
```

Comment:
define this in a const file and import it, otherwise right here it will define the object on every re-render